Assets/Scripts/Game/Controllers/LootController.cs

```diff
@@ -16,6 +19,37 @@ public enum LootType { Module, RareMetal}
 		/// </summary>
 		public int RareMetals { get; private set; }
 
+		/// <summary>
+		/// Data for creating dropped module loot
+		/// </summary>
+		[SerializeField]
+		public LootData moduleDropData;
+
+		/// <summary>
+		/// Data for creating dropped metal loot (appears on map, player can pick it up)
+		/// </summary>
+		[SerializeField]
+		public LootData rareMetalDropData;
+
+		/// <summary>
+		/// Max number of rare metals that will drop when an asteroid is destroyed
+		/// </summary>
+		[SerializeField, PropertyRange(1, 25)]
+		public int maxRareMetalDropAmount;
+
+		/// <summary>
+		/// Max number of modules that will drop when a ship is destroyed
+		/// </summary>
+		[SerializeField, PropertyRange(1, 25)]
+		public int maxModuleDropAmount;
```

Comment:
Do the same for the above variables. [SerializeField] should show the field in the unity editor even if it's private so use properties to access these variables in-code.